### PR TITLE
[Settings v2] Manage MFA - Without Recovery Code

### PR DESCRIPTION
**Pull request by IniZio**
ref DEV-1964
ref DEV-1967
ref DEV-1971
ref DEV-1961

CI: https://github.com/IniZio/authgear-server/pull/1

Dependencies:
- [x] https://github.com/authgear/authgear-server/pull/4784
- [ ] https://github.com/authgear/authgear-server/pull/4786
- [ ] https://github.com/authgear/authgear-server/pull/4790

Demo:

**TOTP**

https://github.com/user-attachments/assets/b4c46e9d-1fd4-414c-a0b7-6b3e9627c382

**Email**

https://github.com/user-attachments/assets/bf1d6ed9-0c2a-47e0-bb3f-d48e1b5a5b00

**OTP**

https://github.com/user-attachments/assets/20333273-8f2d-4802-ba2c-5c297ea30b9c

**Password**

https://github.com/user-attachments/assets/1c37aeaf-cf92-486b-8ca1-6f846c1f1f18

**Change Password**

https://github.com/user-attachments/assets/dc92b22f-ca4d-44d9-b276-73842b66de38

### CODE REVIEW

**Comment by louischan-oursky** [comment]
Please rebase.

File: resources/authgear/templates/de/translation.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
   "copied-button-label": "Kopiert",
   "get-new-recovery-codes-button-label": "Neue Codes erhalten",
   "download-button-label": "Herunterladen",
-  "disconnect-button-label": "Trennen",
+  "remove-biometric-button-label": "Trennen",
```
Comment (louischan-oursky) [inline resources/authgear/templates/de/translation.json:113]: Do not modify v1 translation key

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Comment by louischan-oursky** [comment]
lint failed.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]
1. Remove 2FA authenticator button has no response.
2. Add more than 1 2FA authenticator should not (re-generate recovery code and show).
3. With `authentication.recovery_code.list_enabled=true`, no recovery code related settings is shown.

**Comment by IniZio** [comment]
@louischan-oursky Fixed 1 and 2, but for 3 seems some design is actually missing for regenerating recovery code. 

Asking designer for layout of regenerate action, maybe I handle it in DEV-2006 instead?

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**